js/exporter/excel_format_converter.js

```diff
@@ -85,25 +80,24 @@ var excelFormatConverter = module.exports = {
         }).join("");
     },
 
-    _convertDateFormat: function(format) {
-        format = UNSUPPORTED_FORMAT_MAPPING[format && format.type || format] || format;
+    _convertDateFormat: function(format, isExcelJS) {
```

Comment:
```suggestion
    _convertDateFormat: function(format) {
```
Prepare format in the ExcelCreator

Reply:
Ready